crates/nu-cli/tests/completions/mod.rs

```diff
@@ -363,6 +400,146 @@ fn file_completions() {
     match_suggestions(&expected_paths, &suggestions);
 }
 
+#[derive(Clone)]
+pub struct CustomCommandWithAnyArgs;
+
+impl Command for CustomCommandWithAnyArgs {
+    fn name(&self) -> &str {
+        "list"
+    }
+
+    fn description(&self) -> &str {
+        "Mock def command."
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build("list").rest("args", SyntaxShape::Any, "rest of the arguments")
+    }
+
+    fn run(
+        &self,
+        _engine_state: &EngineState,
+        _stack: &mut Stack,
+        _call: &Call,
+        _input: PipelineData,
+    ) -> Result<PipelineData, ShellError> {
+        todo!()
+    }
+}
+
+#[test]
+fn custom_command_completions() {
```

Comment:
Not a huge deal, but I'd prefer a more descriptive name, like maybe `custom_command_rest_args` or something

Reply:
Me too, I did have a longer name in mind, but only discovered that I pushed the simpler name after the PR checks had started: Then I thought it were not worth it to change if that meant restarting the checks :D

Reply:
Haha, makes sense